src/main/java/com/puppycrawl/tools/checkstyle/meta/JavadocMetadataScraper.java

```diff
@@ -603,13 +617,18 @@ private static String getPackageName(String filePath) {
         return Collections.unmodifiableMap(MODULE_DETAILS_STORE);
     }
 
+    /** Reset the module detail store of any previous information. */
+    public static void resetModuleDetailsStore() {
+        MODULE_DETAILS_STORE.clear();
```

Comment:
Tests are writing to a global static field. Nothing ensure this field is cleared, so this reset method is there to ensure the field is cleared.